src/native/LexFloatClient.h

```diff
@@ -232,16 +232,23 @@ LEXFLOATCLIENT_API int LF_CC GetFloatingClientMeterAttributeUses(CSTRTYPE name,
     RETURN CODES: LF_OK, LF_E_BUFFER_SIZE
 */
 LEXFLOATCLIENT_API int LF_CC GetFloatingClientLibraryVersion(STRTYPE libraryVersion, uint32_t length);
+
 /*
-    FUNCTION: RequestFloatingLicense()
+    FUNCTION: RequestOfflineFloatingLicense()
+
+    PURPOSE: Sends the request to lease the license from the LexFloatServer for offline usage.
 
-    PURPOSE: Sends the request to lease the license from the LexFloatServer.
+    The maximum value of lease duration is configured in the config.yml of LexFloatServer 
+
+    PARAMETERS:
+    * leaseDuration - value of the lease duration.
```

Comment:
@muneebkq Why is the leaseDuration property added as param for RequestFloatingLicense(). There seems to be other docs mistakes as well. Please check the docs again.

Reply:
jsdoc modified